### PR TITLE
Align credentials for GoogleKubernetesEngine, CloudConvert, GoogleSafeBrowsingV2

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [Align credentials for GoogleKubernetesEngine, CloudConvert, GoogleSafeBrowsingV2](https://jira-hq.paloaltonetworks.local/browse/CIAC-6931)

## Description
Adds the option to use credentials for the following integrations:
- CloudConvert
- GoogleSafeBrowsingV2
- GoogleKubernetesEngine

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py [hunk 1]
```diff
@@ -300,10 +306,10 @@ def gcloud_clusters_list_command(client: ClusterManagerClient, project: str, zon
     raw_response_msg: Message = client.list_clusters(project_id=project,
                                                      zone=zone,
                                                      timeout=API_TIMEOUT)
-    raw_response_dict: dict = MessageToDict(raw_response_msg)
+    raw_response_dict: dict = proto.Message.to_dict(raw_response_msg)
```
Comment (JudahSchwartz) [inline Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py:309]: check this returns all expected fields in context output

File: Packs/GoogleSafeBrowsing/Integrations/examples [hunk 2]
```diff
@@ -1 +0,0 @@
-!url url="http://testsafebrowsing.appspot.com/apiv4/ANY_PLATFORM/MALWARE/URL/" using=GoogleSafeBrowsing_instance_1
```
Comment (JudahSchwartz) [inline Packs/GoogleSafeBrowsing/Integrations/examples:1]: why delete this?

**Review by JudahSchwartz — APPROVED** [review]
looks good. lets check k8s thoroughly that know bugs were introduced

File: Packs/GoogleSafeBrowsing/Integrations/examples (hunk 2 above)
Comment (jlevypaloalto) [reply Packs/GoogleSafeBrowsing/Integrations/examples:1]: the build thought it was an integration and got all muddled, i put it in its right place

**Comment by jlevypaloalto** [comment]
This PR must be force-merged as we have no working API key for Cloud Convert.

**Pull request merged**